README.md

```diff
@@ -29,10 +29,12 @@ An Ansible toolkit for Cosmos networks. It allows your control node to:
 Set the appropriate variables in `inventory.yml` and run:
 
 ```
-ansible-playbook gaia.yml -i inventory.yml
+ansible-playbook gaia.yml -i inventory.yml -e 'target=SERVER_IP_OR_DOMAIN'
 ```
 
-- Use the `--extra-vars` option to override the default variables on the command line.
+For multi-node inventory files, don't specify `target` but instead edit the file and replace the existing domains with your server IPs/domains.
+
+- Use the `--extra-vars` or `-e` option to override the default variables on the command line.
 - See the [examples](examples/) for more command, playbook, and configuration options.
 - Visit the [Cosmos testnets repo](https://github.com/cosmos/testnets) for more information.
 
```

Comment:
There is a table with commonly used variables that should be updated with this `target` one.
Eventually, this table should move to a wiki or separate doc.